lib/elixir/lib/list.ex

```diff
@@ -633,21 +633,7 @@ defmodule List do
     [other]
   end
 
-  @doc """
-  Zips corresponding elements from each list in `list_of_lists`.
-
-  The zipping finishes as soon as any list terminates.
-
-  ## Examples
-
-      iex> List.zip([[1, 2], [3, 4], [5, 6]])
-      [{1, 3, 5}, {2, 4, 6}]
-
-      iex> List.zip([[1, 2], [3], [5, 6]])
-      [{1, 3, 5}]
-
-  """
-  @spec zip([list]) :: [tuple]
+  @doc deprecated: "Use Enum.zip/1 instead"
```

Comment:
I think it could be nice to delete the existing implementation and delegate to `Enum.zip/1`  :)

Reply:
Makes sense, I was hesitant since it will now support a list of enumerables and not just a list of lists... but that's not breaking, will go for it.

Reply:
Done in [208f5c2](https://github.com/elixir-lang/elixir/pull/13767/commits/208f5c2bf43dfd7b897cc9ba4d4e8a5df31187ba)

Reply:
That's a good point, maybe it wasn't a great idea. Let's see what people think